Sources/MenuAnimator.swift

```diff
@@ -385,13 +432,19 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
                 }
                 
             case .changed:
+                guard let transitionContext = transitionContext else {
```

Comment:
actually, yes, we don't have an initialized `transitionContext` in `.began` state. That's ok. So, we might refactor this thing later if we found another way for getting initialized transition context.